frontend/src/entity-models/user-login-model.ts

```diff
@@ -0,0 +1,4 @@
+export interface UserLoginModel {
+    email: string,
+    password: string
+}
```

Comment:
eof fix pls

Reply:
> eof fix pls

Idk why new lines are not shown in code. I have 'em in my VScode, but they are not shown here